plugins/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageServiceTests.java

```diff
@@ -47,13 +65,56 @@ GoogleCredential getDefaultCredential() throws IOException {
             }
         };
         assertSame(cred, service.getCredential("default"));
+
+        service.new DefaultHttpRequestInitializer(cred, null, null);
```

Comment:
Is this a typo? Not sure how this compiles...

Reply:
It's a non-static inner class, that's the syntax for constructing an instance of an inner class that is tied to an instance of the outer class.

Reply:
Ahhh, it's been a long time since I have seen this syntax...